app-ui-catalog/src/main/java/app/k9mail/ui/catalog/CatalogContent.kt

```diff
@@ -0,0 +1,66 @@
+package app.k9mail.ui.catalog
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.k9mail.core.ui.compose.common.DevicePreviews
+import app.k9mail.core.ui.compose.designsystem.atom.Surface
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.ui.catalog.items.buttonItems
+import app.k9mail.ui.catalog.items.colorItems
+import app.k9mail.ui.catalog.items.imageItems
+import app.k9mail.ui.catalog.items.selectionControlItems
+import app.k9mail.ui.catalog.items.themeHeaderItem
+import app.k9mail.ui.catalog.items.themeSelectorItems
+import app.k9mail.ui.catalog.items.typographyItems
+
+@Composable
+fun CatalogContent(
+    catalogTheme: CatalogTheme,
+    catalogThemeVariant: CatalogThemeVariant,
+    onThemeChange: () -> Unit,
+    onThemeVariantChange: () -> Unit,
+    contentPadding: PaddingValues,
+    modifier: Modifier = Modifier,
+) {
+    Surface {
+        LazyVerticalGrid(
+            columns = GridCells.Adaptive(300.dp),
```

Comment:
To me it doesn't feel or look great to have more than one column. I'd only set a maximum width for the content area.

Reply:
Setting a maximum width requires additional work, as it doesn't work out of the box.

Reply:
I'll add another approach with the onboarding screen and could apply that to the catalog too.